### PR TITLE
Add validation email input

### PR DESCRIPTION
**Pull request by ManarOmari**
add validation for email input in a form to be lowecase

### CODE REVIEW

File: vailidation-form.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+submitBtn.addEventListener('click', () => {
+  const emailVal = document.getElementById('mail').value;
+
+  if (!validateEmail(emailVal)) {
+    window.alert(
+      'Please enter email as lower case eg. manar.qasem732016@gmail.com',
+    );
+  }
+});
```
Comment (muneebulrehman) [inline vailidation-form.js]: The error message should be near the submit button and not the alert popup. Kindly change this. 
Also, the validation is not working. The error alert shows even if I enter the email in all lower case.
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @ManarOmari , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Your layout for the application in the desktop mode is breaking. There is a lot of space on the right side of your application on bigger displays. It might be because you used fixed width for larger devices. However the case, you should fix it immediately, even though i am marking it as optional. Here is a screenshot of your current app._**
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: vailidation-form.js (hunk 1 above)
Comment (ManarOmari) [reply vailidation-form.js]: solved

File: index.html [hunk 2]
```diff
@@ -401,19 +407,20 @@ <h1> Contact me</h1><br>
...
             <li>
-              <input type="email" name="email"  class="input" placeholder="Email" required  pattern=".+@globex\.com">
+              <input type="email" name="email"  class="input" placeholder="Email" id="mail" value="" required >
             </li>
             <li>
               <textarea name="msg" placeholder="Write your message here" class="textArea input" required maxlength="500"></textarea>
             </li>
             <li>
               <button type="button" class="btn">Get It Touch</button>
+              <label for="text" class="txtt"></label>
```
Comment (leonardodiasb) [inline index.html:418]: - [x] Kindly change the position where your error message appears. It should not move the submit button.
- [OPTIONAL] Your desktop version is not centered in some screen sizes. Kindly review the width of your page to fit in all screen sizes.
- [OPTIONAL] Your form is not properly centered. Kindly review that.

Your design: 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @ManarOmari , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ManarOmari) [reply index.html:418]: solved

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ManarOmari   

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- error message works as expected
- email check ✔️ 
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**